### ISSUE TITLE
[BUG/ISSUE] EDGAR v6 CH4 rice emissions are missing

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#1538]
In the CH4 simulation, rice emissions are extremely low starting in 13.4.0 when EDGAR v6 emissions were introduced (replacing EDGAR v4.3.2). 

```
###################################################################################
### Emissions totals for species CH4 [Tg]                                       ###
### Ref = CH4_13.3.3; Dev = CH4_13.4.0                                          ###
###################################################################################
                                    Ref                 Dev     Dev - Ref    % diff
CH4 BiomassBurn    :          19.412779           19.412779      0.000000     0.000
CH4 Coal           :          31.352809           30.415008     -0.937802    -2.991
CH4 Gas            :          24.919700           14.307448    -10.612253   -42.586
CH4 Lakes          :           0.000000            0.000000      0.000000       nan
CH4 Landfills      :          30.891649           36.385579      5.493931    17.785
CH4 Livestock      :         116.916247          120.381024      3.464778     2.963
CH4 Oil            :          42.265931           24.677758    -17.588174   -41.613
CH4 OtherAnth      :          24.629235           53.667464     29.038229   117.901
CH4 Rice           :          37.789759            0.480895    -37.308864   -98.727
CH4 Seeps          :           1.598907            1.598907      0.000000     0.000
CH4 SoilAbsorb     :         -33.631385          -33.631385      0.000000       nan
CH4 Termites       :          12.016941           12.016941      0.000000     0.000
CH4 Wastewater     :          37.746193           43.828258      6.082065    16.113
CH4 Wetlands       :         147.251804          147.251804      0.000000     0.000
-----------------------------------------------------------------------------------
CH4 Total          :         493.160531          470.792239    -22.368292    -4.536
```

It appears that the entry for CH4_RICE in HEMCO_Config.rc is missing in 13.4.0-14.0.1.

**Comment by msulprizio** [comment]
It appears rice emissions are still included in 13.4.0+, however they're inadvertently lumped with the "Other Anthropogenic" sector. This also explains the increase in "CH4 OtherAnth" seen in the table above.

This confusion was caused by the change in file names from using `IPCC_4C_4D...` in EDGAR v4.3.2 to `AGS` (agricultural soils) in EDGAR v6:

HEMCO_Config.rc for 13.3.3:

```
### Rice ###
0 CH4_RICE__4C_4D        $ROOT/CH4/v2017-10/EDGARv432/EDGAR_v432_CH4_IPCC_4C_4D1_4D2_4D4.generic.01x01.nc                              emi_ch4 1970-2012/1/1/0 C xy kg/m2/s     CH4 11 7 1
```

HEMCO_Config.rc for 13.4.0:

```
0 CH4_OTHER__4C_4D       $ROOT/CH4/v2022-01/EDGARv6/$YYYY/v6.0_CH4_$YYYY_AGS.0.1x0.1.nc              emi_ch4 2000-2018/1-12/1/0 C xy kg/m2/s CH4 - 8 1
```

We can include a fix for this in 14.0.2.

**Comment by yantosca** [comment]
We can close this issue as PR #1540 has been merged into the 14.0.2 development stream.
